typings/index.d.ts

```diff
@@ -1,7 +1,7 @@
-import { Component, StatelessComponent } from 'react';
+import React, { Component, StatelessComponent } from 'react';
 
 declare namespace ReactStaticGoogleMap {
-  interface GoogleMapImageProps {
+  interface GoogleMapImageProps extends Omit<React.HTMLProps<any>, 'size'|'style'> {
```

Comment:
We should also remove src. Since the user doesn't control that.

```suggestion
  interface GoogleMapImageProps
    extends Omit<React.HTMLProps<unknown>, 'size' | 'style' | 'src'> {
```